examples/scripts/project/_common/fork_project.py

```diff
@@ -0,0 +1,115 @@
+""" Example script for creating a copy of an existing project with a new name """
```

Comment:
can you write a usage example in docstrings here please with all flags or some flags (i'm particularly interested in the new project name flag)